x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/action/service/DeleteServiceAccountTokenRequest.java

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.core.security.action.service;
+
+import org.elasticsearch.action.ActionRequest;
+import org.elasticsearch.action.ActionRequestValidationException;
+import org.elasticsearch.action.support.WriteRequest;
+import org.elasticsearch.common.Strings;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.common.io.stream.StreamOutput;
+
+import java.io.IOException;
+import java.util.Objects;
+
+import static org.elasticsearch.action.ValidateActions.addValidationError;
+
+public class DeleteServiceAccountTokenRequest extends ActionRequest {
```

Comment:
This class is currently largely identical to `CreateServiceAccountTokenRequest`. They could share a superclass. I didn't do it because I'd like to have a discussion on whether we should support bulk deletion first.

Reply:
I don't think there's an substantial argument for merging things just because they have similar fields.
We could, it would be fine, but it probably doesn't gain us much

Reply:
I agree, especially in this case, it feels like an accidental similarity.